pkgs/data/icons/catppuccin-papirus-folders/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, papirus-icon-theme
+, gtk3
+, bash
+, getent
+, flavor ? "mocha"
+, accent ? "blue"
+, ...
+}:
+let
+  validAccents = [ "blue" "flamingo" "green" "lavender" "maroon" "mauve" "peach" "pink" "red" "rosewater" "sapphire" "sky" "teal" "yellow" ];
+  validFlavors = [ "latte" "frappe" "macchiato" "mocha" ];
+  pname = "catppuccin-papirus-folders";
+in
+lib.checkListOfEnum "${pname}: accent color" validAccents [ accent ]
+  lib.checkListOfEnum "${pname}: flavor"
+  validFlavors [ flavor ]
+  stdenvNoCC.mkDerivation
+rec {
```

```suggestion
lib.checkListOfEnum "${pname}: accent color" validAccents [ accent ]
lib.checkListOfEnum "${pname}: flavor" validFlavors [ flavor ]

stdenvNoCC.mkDerivation {
```

Indenting could be clearer (I stole this indenting from another random icon theme derivation which did something similar)